specification/ml/infer_trained_model_deployment/MlInferTrainedModelDeploymentResponse.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { double } from '@_types/Numeric'
+import { TrainedModelEntities } from '@ml/_types/TrainedModel'
+
+export class Response {
+  body: {
+    /**
+     * If the model is trained for named entity recognition (NER) tasks, the response contains the recognized entities.
+     */
+    entities?: TrainedModelEntities[]
+    /**
+     * If the model is trained for a text classification task, the response is a predicted label.
+     * For named entity recognition (NER) tasks, the response contains the annotated text output.
+     */
+    predicted_value?: string
```

Comment:
This is a tricky one. All task types will have `prediction_probability` and `predicted_value` fields (if the request was successful) but the actual contents of `predicted_value` depend on the task type

### NER
The comment is correct, you get the annotated text in `predicted_value` and a list of `entities`

### Fill Mask
**predicted_value: string**: The top prediction for replacing the mask token. e.g. "Paris is the capital of [MASK]" will return "France" here

**predicted_value_sequence: string**
The input text sequence with the `[MASK]` token replaced by the predicted_value

**top_classes**: List of TopClassEntry where TopClassEntry is 
```
export class TopClassEntry {
  class_name: string
  class_probability: double
  class_score: double
}
```

### Text Classification
**predicted_value: string**: The predicted class

**top_classes**: List of TopClassEntry 

### Text Embedding
**predicted_value:  array of double** :  The raw text embedding values

### Zero Shot Classification
**predicted_value: string**: The predicted class

**top_classes**: List of TopClassEntry 


### Warning
**warning : string** : If the request failed the reason for the failure

Reply:
Thanks!  I've tried to incorporate all that information in f40c639874ef56f8d061cd6888b887d1b9fcb3fb